### PR TITLE
feat: add wgc feature-flag list command

### PR DESCRIPTION
**Pull request by JivusAyrus**
## TODO

- [ ] Tests or benchmark included
- [ ] Documentation is changed or added on [https://app.gitbook.com/](https://app.gitbook.com/)
- [x] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)

### CODE REVIEW

File: cli/src/commands/feature-flag/commands/list.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        (f) =>
+          ({
+            name: f.name,
+            labels: f.labels.map((l) => joinLabel(l)),
+            isEnabled: f.isEnabled,
+            lastUpdatedAt: f.updatedAt,
+            namespace: f.namespace,
+          }) as OutputFile[number],
+      );
+      await writeFile(options.out, JSON.stringify(output));
```
Comment (StarpTech) [inline cli/src/commands/feature-flag/commands/list.ts:49]: Please use `resolve` here.

File: cli/src/commands/feature-flag/commands/list.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  namespace: string;
+  labels: string[];
+  isEnabled: boolean;
+  lastUpdatedAt: string;
+}[];
+
+export default (opts: BaseCommandOptions) => {
+  const command = new Command('list');
+  command.description('Lists all feature flags in the namespace.');
+  command.option('-n, --namespace [string]', 'The namespace of the feature flags.');
```
Comment (Aenimus) [inline cli/src/commands/feature-flag/commands/list.ts:21]: None of these is required?

File: cli/src/commands/feature-flag/commands/list.ts (hunk 1 above)
Comment (JivusAyrus) [reply cli/src/commands/feature-flag/commands/list.ts:49]: Done

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**